cfme/tests/integration/test_external_auth.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from gevent.timeout import Timeout
+
+from cfme import test_requirements
+from cfme.utils.auth import get_auth_crud
+from cfme.utils.blockers import BZ
+
+pytestmark = [test_requirements.auth]
+
+# External auth specific test cases (e.g. freeipa)
+
+
+@pytest.fixture
+def freeipa_provider():
+    # run this on freeipa03 as to not affect tests running on freeipa01
+    auth_prov = get_auth_crud("freeipa03")
```

Comment:
Can you please fetch this from yamls instead of naming directly as string?

Reply:
Not really, since `auth_providers` are not like normal `providers` in our code base, i.e. we cannot use provider markers for them. We could put this in `cfme.fixtures.authentication` but ultimately we'd have to explicitly specify the key anyway.